overlord/snapstate/booted.go

```diff
@@ -77,9 +74,7 @@ func UpdateRevisions(ovld *overlord.Overlord) error {
 		return fmt.Errorf(errorPrefix+"%s", err)
 	}
 
-	st := ovld.State()
-	st.Lock()
-	installed, err := snapstate.All(st)
+	installed, err := All(st)
```

Comment:
not a blocker, but still wondering why this doesn't just use snapstate.CurrentInfo(st, kernelSnap|osSnap) instead of All